src/lib/dates.ts

```diff
@@ -64,6 +64,11 @@ export const DEFAULT_DURATIONS: DurationOption[] = [
   { label: 'Days', value: 'day' },
 ];
 
+export const DEFAULT_RELATIVE_VARIABLES = [
+  { identifier: 'today', value: '$$NOW', label: 'TODAY' },
+  // TODO: add enabled relative variables here
```

Comment:
Actually we're not that sure we're gonna "hardcode" the relative date variables here (cf discussion with @austil last friday).
So let's keep that on hold and remove this TODO.

However, the playground is already providing a "now" variable, so it should still be testable :)
We can also add another variable (like "the first day of this year") to the playground if we cant to test more cases

Reply:
Not sure to understand, will ask you to explain me dircectly because it's not clear for me